src/current/cockroachcloud/managing-access.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Managing Access (Authorization) in CockroachDB Cloud
+title: Managing Users, Roles, and Service Accounts in CockroachDB Cloud
```

Comment:
This title change was not showing up on the sidebar... they mismatch, perhaps that is on purpose?